### PR TITLE
Deleting all unused Service

### PR DESCRIPTION
**Pull request by lomamech**
- Deleting all old unused Service of Bhima 1.x
- Fix error arround Test E2E for simple voucher
- Deleting some old units who is not used in Bhima 2.x

@sfount @jniles

### CODE REVIEW

**Comment by jniles** [comment]
@lomamech, merci pour une grande effort.  Je vais le tester demain matin, et si ca passe, je vais le melanger.

**Comment by jniles** [comment]
@lomamech, il y a des erreurs sur ta branch, relatif a `locations.js`.

[protractor.log.txt](https://github.com/IMA-WorldHealth/bhima-2.X/files/229460/protractor.log.txt)

**Comment by lomamech** [comment]
Fix error error arround locations Updating (Country, Province, sector, village)

@jniles

File: server/controllers/locations.js [hunk 1]
```diff
@@ -517,9 +513,9 @@ exports.update.province = function updateProvince(req, res, next) {
 
   var data = convert(req.body);
 
-  db.exec(sql, [data, uid])
+  db.exec(sql, [data, req.params.uuid])
```
Comment (jniles) [inline server/controllers/locations.js:516]: This is really helpful!  Thanks!

**Comment by jniles** [comment]
LTGM.  All tests pass!

**Pull request merged**